web/src/routes/login.svelte

```diff
@@ -11,7 +11,7 @@
         if (user && user.twitterUsername === null) {
             localStorage.setItem('initial-login-seller', "1");
         }
-        goto("/auctions");
+        goto("/stall");
```

Comment:
we should go to `/stall/<nym>`, not to `/stall`

Reply:
this part was problematic since the user `nym` does not exist yet therefore I solved by just resolving `stall` to the users own stall